### PR TITLE
Provide Qodana for Go basic documentation

### PR DESCRIPTION
**Pull request by tiulpin**

### CODE REVIEW

File: topics/linters.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ The [](qodana-js.md) linter is based on [WebStorm](https://www.jetbrains.com/web
 * JavaScript and TypeScript language inspections from WebStorm
 * Third-party license verification using <a href="license-audit.xml">License audit</a>
 
+## Go
+
+The [](qodana-go.md) linter is based on [GoLand](https://www.jetbrains.com/go) provides Go language inspections from GoLand.
```
Comment (rillig) [inline topics/linters.md]: * missing 'and' before 'provides'
* I like the word 'provides' over 'supports', we should replace that in the other sections as well.

File: topics/qodana-go-docker-readme.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+<topic xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/stardust/topic.v2.xsd"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="qodana-go-docker-readme"
+       title="Qodana for Go">
+
+    <var name="linter" value="Qodana for Go"/>
+    <var name="linter-shell" value="qodana-go"/>
+    <var name="linter-page-url" value="qodana-go.md"/>
+    <var name="docker-tech-guide-page-url" value="qodana-go-docker-techs.xml"/>
+    <var name="ide" value="GoLand"/>
+    <var name="code-inspection-ide-help-url" value="https://www.jetbrains.com/help/webstorm/?Code_Inspection"/>
```
Comment (rillig) [inline topics/qodana-go-docker-readme.xml:14]: `/goland/` instead of `/webstorm/`

File: topics/qodana-go-docker-techs.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+<!--suppress TopicContentShortcutHardcodedInspections -->
+<topic xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/stardust/topic.v2.xsd"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="qodana-go-docker-techs"
+       title="Qodana for JS">
+
+    <var name="linter" value="Qodana for Go"/>
+    <var name="linter-shell" value="qodana-go"/>
+    <var name="ide" value="GoLand"/>
+    <var name="docker-general-guide-page-url" value="qodana-go-docker-readme.xml"/>
+    <var name="code-inspection-profiles-ide-help-url" value="https://www.jetbrains.com/help/webstorm/?Customizing_Profiles"/>
```
Comment (rillig) [inline topics/qodana-go-docker-techs.xml:13]: `/goland/` instead of `/webstorm/`

File: topics/qodana-go-docker-techs.xml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            <include src="lib_qd.xml" include-id="docker-config-change-heap-size"/>
+
+        </chapter>
+        <chapter id="Log+INFO+messages+to+STDOUT" title="Log INFO messages to STDOUT">
+
+            <include src="lib_qd.xml" include-id="docker-config-log-info-to-stdout"/>
+
+        </chapter>
+
+        <chapter id="Use+a+different+idea-properties-file"
```
Comment (rillig) [inline topics/qodana-go-docker-techs.xml:53]: The mixture of `+` and `-` is strange. Maybe a better `id` would just be `idea-properties`, to make it immune to future wording changes.

File: topics/qodana-go-docker-techs.xml [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+        <chapter id="Turn+off+user+statistics" title="Turn off user statistics">
+
+            <include src="lib_qd.xml" include-id="docker-config-disable-statistics"/>
+
+        </chapter>
+
+        <chapter id="Manage+plugins" title="Manage plugins">
+
+            <include src="lib_qd.xml" include-id="docker-config-manage-plugins" use-filter="empty,js"/>
```
Comment (rillig) [inline topics/qodana-go-docker-techs.xml:68]: `empty,go` instead of `empty,js`

If the intention is to say that GoLand comes bundled with JavaScript and TypeScript, this should be made clear in the `docker-config-manage-plugins` chunk.

File: topics/qodana-go.md [hunk 6]
```diff
@@ -3,46 +3,29 @@
 [![official project](https://jb.gg/badges/official-flat-square.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 
 <note>
-<p>
+    <p>
```
Comment (rillig) [inline topics/qodana-go.md:6]: Why this change in indentation?

**Review by smth-to-smb — APPROVED** [review]

File: topics/qodana-go-docker-techs.xml (hunk 4 above)
Comment (smth-to-smb) [reply topics/qodana-go-docker-techs.xml:53]: Unified the minus signs for +.

File: topics/qodana-go-docker-techs.xml (hunk 5 above)
Comment (smth-to-smb) [reply topics/qodana-go-docker-techs.xml:68]: Added the go condition to the chunk.
